src/use-local-storage/index.spec.tsx

```diff
@@ -1,9 +1,8 @@
 import { act, renderHook } from '@testing-library/react-hooks'
-import localStorage from './localStorage'
 import useLocalStorage from './index'
 
 afterEach(() => {
-  localStorage.clear()
+  window.localStorage.clear()
```

Comment:
We are running these tests on jest, which has `window`. So this change will have no effects on tests.